addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -13,13 +13,12 @@
   },
   "private": true,
   "dependencies": {
-    "angular2": "2.0.0-beta.9",
-    "es6-promise": "^3.0.2",
```

Comment:
Was `es6-promise` rolled into some other polyfill? I still see it in https://github.com/angular/angular/blob/master/package.json#L35